diff --git a/src/qtil_file.cpp b/src/qtil_file.cpp
index 4aa4ddf..00e4bc0 100644
--- a/src/qtil_file.cpp
+++ b/src/qtil_file.cpp
@@ -74,7 +74,7 @@ QString File::readText() const
     QFile file(path());
     if (file.exists() && file.open(QIODevice::ReadOnly | QIODevice::Text)) {
         QTextStream in(&file);
-        in.setCodec("UTF-8");
+        in.setEncoding(QStringConverter::Utf8);
         data = in.readAll();
         file.close();
     }
@@ -86,7 +86,7 @@ bool File::writeText(const QString &data) const
     QFile file(path());
     if (file.open(QIODevice::WriteOnly | QIODevice::Text)) {
         QTextStream out(&file);
-        out.setCodec("UTF-8");
+        out.setEncoding(QStringConverter::Utf8);
         out << data;
         file.close();
         return true;
